### DISCUSSION TITLE
GLTF is'nt visible. No errors either.

### DISCUSSION THREAD

**Discussion by garyiskidding** [google/model-viewer#3163] [Q&A]
[cattle_gltf.zip](https://github.com/google/model-viewer/files/7966136/cattle_gltf.zip)

Adding such gltf(attached as zip) files shows no errors, but the model is not shown either. 

Have checked this model and appears to be valid, and working on other gltf players (like https://sandbox.babylonjs.com/) correctly.

Please advice why does modelviewer fail to show a valid gltf.

**Comment by samaneh-kazemi** [comment]
Hi, can you please share more details on this? Are you using the editor or you're embedding model-viewer in your code?

**Reply by garyiskidding** [reply]
Oh sure. Let me add all the info. Sorry I should have done it to begin with. I tried the editor, and embedding modelviewer, its the same result : a blank screen
![Screenshot 2022-01-30 204115](https://user-images.githubusercontent.com/10496265/151707867-9171bec7-4f42-4129-8e97-82e81cae1e4a.png)

**Reply by garyiskidding** [reply]
The details from "WARNINGS"
![image](https://user-images.githubusercontent.com/10496265/151707895-dd729f98-4d96-4cac-9840-9f8c23b13dc5.png)

**Reply by garyiskidding** [reply]
Same file works like a charm on sandbox.babylon.js : 
![image](https://user-images.githubusercontent.com/10496265/151707924-968264e0-4ef3-4d78-b1ed-6f90649c5715.png)

**Comment by garyiskidding** [comment]
Requesting someone to please take a look at this issue and advise.

**Comment by samaneh-kazemi** [comment]
Hi @garyiskidding, I uploaded your files in model-viewer editor and I see many errors in the js console. I attached it here. Most likely you exported your gltf with incorrect paths. I reacommend you convert your model to GLB to avoid errors like this. 
![Screen Shot 2022-02-02 at 9 34 50 AM](https://user-images.githubusercontent.com/3915613/152208380-681526b6-fe1f-46d5-a7cc-fc63b3c09c92.png)

**Reply by garyiskidding** [reply]
Thanks for your reply @samaneh-kazemi  ,
[bear_13.zip](https://github.com/google/model-viewer/files/8065959/bear_13.zip)

We have used GLB(attached as zip) and have tried to make sure that it is a clean export. This is file, we do not have warnings for THREE.js, etc.

The file again works really well on babylon : 
![image](https://user-images.githubusercontent.com/10496265/153995771-b6b7de54-3feb-46fe-81a0-5dc3d6dd8973.png)


But just does not show up on modelviewer editor:
![image](https://user-images.githubusercontent.com/10496265/153995884-666f2b78-9feb-42ab-9abe-c409f823da28.png)

Notice that the file is free of any THREE.js warnings.

Please help. This is just one example of multiple clean GLB files that just dont show up on modelviewer but work really nicely in babylon.

**Reply by samaneh-kazemi** [reply]
Hi @garyiskidding, I just tried uploading your model into our editor and I got validator warnings. Please update your model to clear the warnings and try again.
 
![Screen Shot 2022-02-15 at 9 29 10 AM](https://user-images.githubusercontent.com/3915613/154121860-37c7adb2-59b3-4ef0-81c0-505c9d0c9d1e.png)

**Reply by garyiskidding** [reply]
We have been trying multiple export plugins in both maya and blender, but we always get this warning, even after a clean export from them. 

After some more research, on the official Khronos Gltf exporter github, we found that this warning should be harmless and can be ignored. Here is the link to the official gltf exporter thread : 

https://github.com/KhronosGroup/glTF-Validator/issues/133#issuecomment-892958382

And here is the suggestion : 
![Screenshot 2022-03-03 at 11 26 22 AM](https://user-images.githubusercontent.com/10496265/156505240-325d3aca-6334-4b1e-aa1e-37a300f58a58.png)

From this, it appears that the file should be considered valid and should render in ModelViewer, just like Babylon.js.

Some directions or documentaiton on how to get rid of this warning in blender/maya would be very helpful

**Reply by elalish** [reply]
I think the bug you describe might be this issue: https://github.com/google/model-viewer/issues/3189 Please read it through as it may give some insight into how to avoid this problem. However, I believe we should fix this bug to make things easier. Please leave feedback there if you notice anything we haven't thought of.

**Reply by garyiskidding** [reply]
Thank you @elalish . Will go through it.
